### PR TITLE
Updating Xilinx BSP to 2021.2

### PR DESCRIPTION
**Pull request by ricardosalveti**
Still missing u-boot and kernel updates, but that can follow in another PR.

Tested with https://github.com/ricardosalveti/lmp-manifest/commits/master.

### CODE REVIEW

**Review by angolini — APPROVED** [review]
LGTM

File: meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx/uz3eg-iocc/pm_cfg_obj.c [hunk 1]
```diff
@@ -3,6 +3,7 @@
 * SPDX-License-Identifier: MIT
 ******************************************************************************/
 
+
```
Comment (Tim-Anderson) [inline meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx/uz3eg-iocc/pm_cfg_obj.c:6]: Please remove extra newline

Comment (ricardosalveti) [reply meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx/uz3eg-iocc/pm_cfg_obj.c:6]: Oh, this is just a file copy from the new pmu-firmware, didn't want to change what is coming from the xilinx tooling.

Basically I just built the new pmu firmware and copied over the new files.

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Pull request merged**